scapy/route.py

```diff
@@ -132,8 +132,6 @@ def ifadd(self, iff, addr):
 
 
     def route(self,dest,verbose=None):
-        if isinstance(dest, list) and dest:
-            dest = dest[0]
```

Comment:
This does not exist on `route6.py`, is not specified anywhere and has a weird behavior.
I think it's better to error when a list is provided, than doing that trick